doc/rst/technotes/libraries.rst

```diff
@@ -384,6 +416,68 @@ For example:
    Make sure your Python functionality is also complete before calling this
    function.
 
+.. note::
+
+  If you are taking advantage of the generated ``__init__.py`` initializer
+  file to import the output directory as a package, you do not need to call
+  ``chpl_cleanup()`` yourself because it is already registered to be called
+  at program exit. The generated initializer is explained below.
+
+Python Init File
+----------------
+
+A simple ``__init__.py`` file is generated in the output directory along with
+the Python module. It looks roughly like the following:
+
+.. code-block:: python
+
+  import atexit
+
+  #
+  # Here directoryName is the name of the directory containing your Python
+  # module, and moduleName is the name of your Python module.
+  #
+  from directoryName.moduleName import *
+
+  atexit.register(moduleName.chpl_cleanup)
+
+The initializer file makes working with generated Python code easier than
```

Comment:
This sentence seems unnecessary, but it does make me want to use this feature! :D